dovpanda/config.py

```diff
@@ -10,11 +10,14 @@
 RESTRICTED_DIRS = [PANDAS_DIR, CURDIR]
 # pandas mathods
 READ_METHODS = [method for method in dir(pandas) if 'read' in method]
+WRITE_TEXT_METHODS = ['DataFrame.to_csv', 'DataFrame.to_json']
 DF_CREATION = READ_METHODS + ['DataFrame']
 SERIES_CREATION = READ_METHODS + ['Series.__init__']
 GET_ITEM = ['DataFrame.__getitem__', 'Series.__getitem__',
             'core.indexing._NDFrameIndexer.__getitem__', 'core.indexing._LocationIndexer.__getitem__']
 
+SOURCE_FUNC_NAME = 'source_func_name'
+DOVPANDA_INFO_DICT = '_dovpanda'
```

Comment:
I don't see the need to put those as config constants

Reply:
I was on the impressions that 'source_func_name' would be used a lot, since this is actually the way to find out which function called a specific hook (in cases several functions do so)

Reply:
It will be used a lot, but I don't see a reason to make a const for it. It is a dictionary key, not a value that may change in the future.

Reply:
I can't say that I agree, because if it will be used in even 2-3 places in the code, and we'll change it in the future, then we should not write the specific string in the code.
But ok, I'll delete it from the config file.